Bio/SearchIO/BlatIO.py

```diff
@@ -653,7 +656,8 @@ def _build_row(self, qresult):
                 line.append(hsp.hit_gap_num)
 
                 # check spans
-                assert hsp.query_span_all == hsp.hit_span_all
+                eff_query_spans = [blocksize_multiplier * s for s in hsp.query_span_all]
+                assert hsp.hit_span_all == eff_query_spans
```

Comment:
Good point. Looking at other parts of SearchIO, I think I was quite deliberate in my use of `assert`s. Perhaps it's a good idea to check this in other places as well.